### PR TITLE
Swift convenience

### PR DESCRIPTION
**Pull request by ndis1**
some convenience methods to alleviate the casting we've had to do to use datasource in swift

remove example pod files

also updates the example project and renames some things to play nice with frameworks

### CODE REVIEW

**Comment by ndis1** [comment]
test target not linking for commandline only

**Comment by codecaffeine** [comment]
@ndis1 there are multiple Xcode projects in Example/
```sh
➜  Example git:(swift_convenience) pod install
Updating local specs repositories

Analyzing dependencies
[!] Could not automatically select an Xcode project. Specify one in your Podfile like so:

    xcodeproj 'path/to/Project.xcodeproj'
```

**Comment by codecaffeine** [comment]
LGTM :+1:

**Comment by ndis1** [comment]
@codecaffeine reverted bridgeable per MTM conversation


I'd like to start using this soon pending other input @rmigneco @jgrandelli

File: Pod/Classes/SwiftConvenience.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+//
+//  Created by Nick DiStefano on 2/1/16.
+//
+//
+
+import Foundation
+
+
+extension URBNDataSourceAdapter {
+    public func registerCell<CellElement: UIView, DataElement: AnyObject>(identifier: String = "\(CellElement.self)", configurationBlock: (CellElement, DataElement, NSIndexPath) -> ()) {
```
Comment (rmigneco) [inline Pod/Classes/SwiftConvenience.swift]: :+1:

**Comment by rmigneco** [comment]
Big fan of this @ndis1

File: Example/URBNDataSource/ViewControllers/Code CollectionView/URBNSwiftAccordionTableViewController.swift [hunk 2]
```diff
@@ -43,15 +44,10 @@ class URBNSwiftAccordionTableViewController: UITableViewController {
...
-            let object = object as? String else {
-                return
-            }
-            
-            cell.textLabel?.text = object
+        adapter.registerCell { (cell: UITableViewCell, object: NSString, indexPath) in
+            cell.textLabel?.text = object as String
         }
-
+
```
Comment (jgrandelli) [inline Example/URBNDataSource/ViewControllers/Code CollectionView/URBNSwiftAccordionTableViewController.swift]: Thanks for checking this in.

Comment (jgrandelli) [inline Example/URBNDataSource/ViewControllers/Code CollectionView/URBNSwiftAccordionTableViewController.swift]: :trollface:

**Pull request merged**